spec/models/subscriber_list_spec.rb

```diff
@@ -29,6 +29,46 @@
     end
   end
 
+  describe "validations" do
+    subject { FactoryGirl.build(:subscriber_list) }
```

Comment:
Would the tests be more independently readable if `subject` were changed to `let(:subscriber_list)`?